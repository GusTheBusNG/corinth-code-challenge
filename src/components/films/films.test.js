import React from 'react';
import { create } from 'react-test-renderer';

import Films from './';

const createFilms = () => create(<Films films={['1']} />);

describe('Films', () => {
  describe('render', () => {
    it('renders correctly', () => {
      expect(createFilms().toJSON()).toMatchSnapshot();
    })
  })
})
