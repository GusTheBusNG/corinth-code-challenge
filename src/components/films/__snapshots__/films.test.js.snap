// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Films render renders correctly 1`] = `
<div
  className="films"
>
  <img
    alt="film"
    className="film-poster"
    src="https://res.cloudinary.com/dnglix0tu/image/upload/f_auto/v1/LifeWay-star-wars/movies/1?_a=ATAMhAA0"
  />
</div>
`;
